### ISSUE TITLE
Save issue as a draft

### ISSUE THREAD

**Issue by fastengineer** [isaacs/github#1759]
While I'm writing an issue. The electric gone! And one page which I write just gone!

This is an infamous situation. Every web app have the auto saving feature today!

**Comment by dpikt** [comment]
Possible duplicate of https://github.com/isaacs/github/issues/1648?

**Comment by fastengineer** [comment]
Yes. But I'm talking about autosaving as draft. But the issue not talking about autosaving. Having an old style draft feature not fixes problem.

**Comment by Mancerrss** [comment]
Just experienced this today. Yes the bug still does exist. Writing issues, comments doesn't auto save.

When a browser crashed, or the person left that certain repo, issue comment thread webpage and already written something, it should be left there.

Frankly, I would suggest a more obvious UI that shows spinning circle and a check to indicate that this comment has already been synced up and is autosaved, in case you wanna get back at it again.

Let us have a page on our profile where we can get back on our Drafted issues, docus and comments too.

**Comment by keyvan-m-sadeghi** [comment]
+1

**Comment by georgeolaru** [comment]
After losing some longer-than-necessary comments by closing the browser accidentally, I started using a Chrome extension ([Typio](https://chrome.google.com/webstore/detail/typio-form-recovery/djkbihbnjhkjahbhjaadbepppbpoedaa)) that help me recover any input data in case of loss. 

Until drafting an issue comes as a default feature to Github, the tool above can get this job done pretty well.

**Comment by aspiers** [comment]
This sounds great - thanks for the recommendation!

**Comment by Arkenan** [comment]
+1 to the importance of this feature. I spent about 45 minutes writing an issue, as I was looking for data elsewhere to put in it. The UI suddenly changed things around, and I accidentally ended up clicking "choose another template", which completely erased my whole issue except from the title. Buttons that potentially have the risk of erasing everything should at least have a prompt, specially if there's no option for auto-saving drafts.

**Comment by rfiedorowicz** [comment]
Ah, I just half of hour of working on the big issue with screens. 
Autosaving would save me now :(

**Comment by aspiers** [comment]
I've found [Typio Form Recovery](https://typiorecovery.github.io/) to be a very useful workaround in general for this kind of issue, although for some reason it seems not to work on GitHub comments.
